src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -75,7 +103,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     }
 
     if(user.isAdmin()){
-      // user is in admin list, give access to admin site
+      // user is in admin list, give access to admin site, and send map as attribute
+      addStats(map);
+      request.setAttribute("map", map);
```

Comment:
I would make the attribute name a bit more descriptive .. "adminStatsMap" or something..

Reply:
Done!